Gruntfile.js

```diff
@@ -15,23 +15,49 @@ module.exports = function(grunt) {
         serverToken: '<%= secrets.serverToken %>'
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Yo',
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Hey',
         src: ['test/*.html']
       }
-    }
+    },
+
+    "postmark-templates": {
```

Comment:
I built postmark-server to be idempotent, which wasn't too hard. The issue I encountered with doing the same for postmark-template is that you need the templateID to update, as I don't think template names are unique identifiers.